<launch>
<node name="imu_node" pkg="segway" type="imu_node.py" output="screen" required="true" launch-prefix="nice -n -20"/>
<node name="motor_node" pkg="segway" type="motor_node.py" output="screen" required="true" launch-prefix="nice -n -20"/>
</launch>
